app/experimenter/experiments/models/nimbus.py

```diff
@@ -19,7 +19,11 @@
 
 class NimbusExperimentManager(models.Manager):
     def launch_queue(self, application):
-        return self.filter(status=NimbusExperiment.Status.REVIEW, application=application)
+        return self.filter(
+            status=NimbusExperiment.Status.DRAFT,
+            publish_status=NimbusExperiment.PublishStatus.APPROVED,
+            application=application,
+        )
```

Comment:
Further along the launch-from-preview flow, I think this condition needs to look for both DRAFT and PREVIEW [a bit like we did for `start_date`](https://github.com/mozilla/experimenter/pull/4946/files#diff-36bd393fce70ef5fc7ad821bc3c232a4b00882f000583f222fc3041f735d3bdbR187-R189):

```python
  Q(status=NimbusExperiment.Status.DRAFT)
  | Q(status=NimbusExperiment.Status.PREVIEW),
```